tensorboard/components/tf_data_selector/storage-utils.ts

```diff
@@ -15,15 +15,15 @@ limitations under the License.
 namespace tf_data_selector {
 
 export function decodeIdArray(str: string): Array<number> {
-  return str.split(',').map(idStr => parseInt(idStr, 36)).filter(Boolean);
+  return str.split(',').map(idStr => parseInt(idStr, 10)).filter(Boolean);
```

Comment:
Why do we remove `0`s from the result array?

(If this is a bug, please fix it in a separate PR.)

Reply:
sounds like a bug. Will do